py/selenium/webdriver/remote/webelement.py

```diff
@@ -539,7 +539,13 @@ def location(self):
     @property
     def rect(self):
         """A dictionary with the size and location of the element."""
-        return self._execute(Command.GET_ELEMENT_RECT)['value']
+        if self._w3c:
+            return self._execute(Command.GET_ELEMENT_RECT)['value']
+        else:
+            rect = self.size.copy()
+            rect.update(self.location)
+            return rect
+
```

Comment:
flake8 is going to fail on this having 2 lines

Reply:
Fixed